config/application.rb

```diff
@@ -54,13 +60,7 @@ class Application < Rails::Application
       batch_config.operation_middleware = proc {}
     end
 
-    config.active_record.belongs_to_required_by_default = true
```

Comment:
did you mean to disable this? It's a new default in Rails since version 6 so it's nice we had it enabled to begin with.

Reply:
no, please re-enable! 

could this have been automatically overriden (eg when running `rails app:update`)? there are many other settings that were also removed... could you have a look?
https://github.com/impactoss/impactoss-server/pull/436/commits/7fd03cc012cda6f9c70583203f509785312c135d#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9

Reply:
I have gone through and restored the config settings, or upgraded where appropriate.
Running the command `app:update` overwrites any config files with new ones, and should instead be skipped when asked to overwrite, so that we can then hand hold it through changes.